Snakefile

```diff
@@ -1,3 +1,17 @@
+GENOME_VERSION = "GRCh38"
+ENSEMBL_VERSION = "96"
+GENEMODEL_VERSION = GENOME_VERSION + "." + ENSEMBL_VERSION
```

Comment:
@bintiibrahim Here are the three fields I think we need to populate from the Ensembl FTP server.

Reply:
Oh, plus the species name, e.g. "homo_sapiens"